app.rb

```diff
@@ -17,62 +18,68 @@
   content_type "application/json"
 end
 
-authenticate_jwt
+authenticate_jwt("/api/*")
 
-post "/submit" do
-  unless params.has_key?("submission_bundle")
-    status 400
-    return {error_message: "Request missing submission_bundle param"}.to_json
-  end
+get "/health" do
+  :ok
+end
 
-  submission_filename = params["submission_bundle"]["filename"]
-  result = Dir.mktmpdir do |dir|
-    submission_path = File.join(dir, submission_filename)
-    FileUtils.mv params["submission_bundle"]["tempfile"].path, submission_path
-    EfilerService.run_efiler_command(get_api_client_mef_credentials, "submit", submission_path)
-  end
+namespace "/api/v0" do
```

Comment:
what is the purpose of `v0` on this namespace? I get that it's a way to version the api, but when will we increment the version?